src/index.html

```diff
@@ -9,6 +9,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logotype">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="navigation">
+        <ul class="nav-list">
```

Comment:
Don't forget to add an empty line between multiline siblings 
```suggestion
      </a>
      
      <nav class="navigation">
        <ul class="nav-list">
```